graphics/src/OBJLoader_TEST.cc

```diff
@@ -69,9 +69,69 @@ TEST_F(OBJLoaderTest, InvalidMaterial)
     common::testing::TestFile("data", "invalid_material.obj");
 
   ignition::common::Mesh *mesh = objLoader.Load(meshFilename);
+
   EXPECT_TRUE(mesh != nullptr);
 }
 
+/////////////////////////////////////////////////
+// This tests opening an OBJ file that has PBR fields
+TEST_F(OBJLoaderTest, PBR)
+{
+  ignition::common::OBJLoader objLoader;
+
+  // load obj file exported by 3ds max that has pbr extension
+  {
+    std::string meshFilename =
+      common::testing::TestFile("data", "cube_pbr.obj");
+
+    ignition::common::Mesh *mesh = objLoader.Load(meshFilename);
+    EXPECT_TRUE(mesh != nullptr);
+
+    const common::MaterialPtr mat = mesh->MaterialByIndex(0u);
+    ASSERT_TRUE(mat.get());
+
+    EXPECT_EQ(math::Color(0.0, 0.0, 0.0, 1.0), mat->Ambient());
+    EXPECT_EQ(math::Color(0.5, 0.5, 0.5, 1.0), mat->Diffuse());
+    EXPECT_EQ(math::Color(1.0, 1.0, 1.0, 1.0), mat->Specular());
+    EXPECT_DOUBLE_EQ(mat->Transparency(), 0.0);
+    EXPECT_TRUE(mat->TextureImage().find("LightDome_Albedo.png") !=
+        std::string::npos);
```

Comment:
```suggestion
    EXPECT_NE(std::string::npos,
      mat->TextureImage().find("LightDome_Albedo.png"));
```

Reply:
fixed. 987028d

Reply:
Why don't you use `EXPECT_NE` instead of `EXPECT_TRUE` ?

Reply:
oops missed that. fixed. 3d82b3c